cmd/deploy.go

```diff
@@ -380,6 +390,12 @@ type deployConfig struct {
 	// Remote indicates the deployment (and possibly build) process are to
 	// be triggered in a remote environment rather than run locally.
 	Remote bool
+
+	// Enable additional integrations
+	Enable []string
```

Comment:
What if we use `extensions` here and to mark it to be disable use a notation with `-` prefix? There is similar approach is used with envs if I am not mistaken.

Enable/Disable sounds too generic to me and might be confusing.